.github/workflows/src/summarize-checks/dump-trigger-metadata.js

```diff
@@ -0,0 +1,55 @@
+// @ts-check
+
+/**
+ * @param {import('@actions/github-script').AsyncFunctionArguments} AsyncFunctionArguments
+ * @returns {Promise<void>}
+ */
+export default async function dumpTriggerMetadata({ context, core }) {
+  core.info(`Event name: ${context.eventName}`);
+  core.info(`Action: ${context.payload.action}`);
+
+  if (
+    context.eventName === "workflow_run" &&
+    context.payload.action === "completed" &&
```

Comment:
Need to ensure action="completed" to safely cast to WorkflowRunCompletedEvent.  I think you were implicitly assuming this already.

Reply:
Yeah, because we are only triggered on `completed` WF in summarize-checks.yaml.